### ISSUE TITLE
Cannot make Windows release / build env for Windows

### ISSUE THREAD

**Issue by rly** [NeurodataWithoutBorders/nwb-guide#597]
https://github.com/NeurodataWithoutBorders/nwb-guide/actions/runs/7761500374/job/21170068492#step:4:639

When running `conda env update --name nwb-guide --file environments/environment-Windows.yml`:

Error:
```
Installing pip dependencies: ...working... ::warning::Pip subprocess error:%0D%0A  Running command git clone --filter=blob:none --quiet https://github.com/catalystneuro/neuroconv.git 'C:\Users\runneradmin\AppData\Local\Temp\pip-install-ssjsf8f4\neuroconv_c9c2bec0d5a24ee88230b86de2ee8a37'%0D%0D%0A  Running command git clone --filter=blob:none --quiet https://github.com/hdmf-dev/hdmf.git 'C:\Users\runneradmin\AppData\Local\Temp\pip-install-ssjsf8f4\hdmf_4b92963d58874798a798e1f1b9fcfe67'%0D%0D%0A  Running command git submodule update --init --recursive -q%0D%0D%0A  error: unknown subcommand: `update-clone'%0D%0D%0A  usage: git submodule--helper <command>%0D%0D%0A%0D%0D%0A  error: subprocess-exited-with-error%0D%0D%0A  %0D%0D%0A  � git submodule update --init --recursive -q did not run successfully.%0D%0D%0A  \u2502 exit code: 129%0D%0D%0A  \u2570\u2500> See above for output.%0D%0D%0A  %0D%0D%0A  note: This error originates from a subprocess, and is likely not a problem with pip.%0D%0D%0Aerror: subprocess-exited-with-error%0D%0D%0A%0D%0D%0A� git submodule update --init --recursive -q did not run successfully.%0D%0D%0A\u2502 exit code: 129%0D%0D%0A\u2570\u2500> See above for output.%0D%0D%0A%0D%0D%0Anote: This error originates from a subprocess, and is likely not a problem with pip.%0D%0D%0A
  Pip subprocess error:
    Running command git clone --filter=blob:none --quiet https://github.com/catalystneuro/neuroconv.git 'C:\Users\runneradmin\AppData\Local\Temp\pip-install-ssjsf8f4\neuroconv_c9c2bec0d5a24ee88230b86de2ee8a37'
  
    Running command git clone --filter=blob:none --quiet https://github.com/hdmf-dev/hdmf.git 'C:\Users\runneradmin\AppData\Local\Temp\pip-install-ssjsf8f4\hdmf_4b92963d58874798a798e1f1b9fcfe67'
  
    Running command git submodule update --init --recursive -q
  
    error: unknown subcommand: `update-clone'
  
    usage: git submodule--helper <command>
  
  
  
    error: subprocess-exited-with-error
  
    
  
    � git submodule update --init --recursive -q did not run successfully.
  
    \u2502 exit code: 129
  
    \u2570\u2500> See above for output.
  
    
  
    note: This error originates from a subprocess, and is likely not a problem with pip.
  
  error: subprocess-exited-with-error
  
  
  
  � git submodule update --init --recursive -q did not run successfully.
  
  \u2502 exit code: 129
  
  \u2570\u2500> See above for output.
  
  
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by CodyCBakerPhD** [comment]
Some kind of issue with the submodule step of the HDMF install, looks like

**Comment by CodyCBakerPhD** [comment]
Conda error is resolved; now similar issue to #598 https://github.com/NeurodataWithoutBorders/nwb-guide/actions/runs/7819886365/job/21333329563
